packages/driver/src/cy/commands/asserting.ts

```diff
@@ -134,13 +132,15 @@ export default function (Commands, Cypress, cy, state) {
 
   Commands.addAll({ type: 'assertion', prevSubject: true }, {
     should () {
+      // Casted to `any` to pass all arguments
```

Comment:
Nit: The past tense of cast is cast. My lint spellchecker doesn't flag 'casted' as an unknown word, though 🤷‍♂️

Reply:
According to [this answer](https://github.com/cypress-io/cypress/issues/2372), **casted** was used in Middle English and can be used when it's used for cast in programming context. 

But I changed all `casted` to `cast` anyway.

Reply:
How do I get this done?

Reply:
@finda841 In this PR, there's nothing you can do because I've finished all the suggested changes. If you want to help develop Cypress, you can find [`ready-for-work` issues here](https://github.com/cypress-io/cypress/labels/stage%3A%20ready%20for%20work).